jwst/datamodels/schemas/irs2.schema.yaml

```diff
@@ -29,3 +29,13 @@ allOf:
         datatype: float32
       - name: beta_3
         datatype: float32
+    dq_table:
+      title: Reference file for IRS2 refpix correction
```

Comment:
The title for this extension/table should refer to that table, e.g. "Table of Data Quality information"

Reply:
The title above for `irs2_table` should also replace "Reference file" with "Table"